app/src/main/AndroidManifest.xml

```diff
@@ -89,7 +88,6 @@
             android:screenOrientation="portrait" />
         <activity
             android:name=".views.HistoryScanActivity"
-            android:label="History"
```

Comment:
It doesn't reappear in the string file ?

Reply:
@teolemon The string is equivalent to "R.string.scan_history_drawer" which is translatable . Hence label has been removed and title has been set using "R.string.scan_history_drawer"  in HistoryScanActivity .